src/container/srv_container.c

```diff
@@ -4537,3 +4537,120 @@ ds_cont_destroy_orphan(struct cont_svc *svc, uuid_t uuid)
 
 	return rc;
 }
+
+int
+ds_cont_iterate_labels(struct cont_svc *svc, rdb_iterate_cb_t cb, void *arg)
+{
+	struct rdb_tx	tx;
+	int		rc;
+
+	ABT_rwlock_rdlock(svc->cs_lock);
+	rc = rdb_tx_begin(svc->cs_rsvc->s_db, svc->cs_rsvc->s_term, &tx);
+	if (rc == 0) {
```

Comment:
miss ABT_rwlock_unlock(svc->cs_lock); if rc != 0 ?

Reply:
It is the wrong oder between lock and tx_begin. I will fix it. Thanks!